docs/source/models/cause.rst

```diff
@@ -4,17 +4,119 @@
 
 .. contents:
+   :local:
+
+
+What is a "cause"?
+------------------
+
+Short answer: disease
+
+More nuance: “cause of death” as might be included as the bottom line of the
+top half of a death certificate. And also analogous causes of nonfatal health
+loss, too.
+
+.. image:: death_certificate.png
+
+
+Learning objectives
+-------------------
+
+After reading this chapter, learners should be able to:
 
-What is a cause?
+1. Develop an understanding of how GBD, literature, and experts think about
+   a cause.
+2. Build internally consistent models which are sufficiently complex given
+   larger modeling goals.
+
+   a. Models that are as simple as possible, but no simpler.
+   b. Models that agree with withheld data.
+   c. Models that captures the outcomes of interest. (Which is really the same
+      as “but no simpler” in (a))
+
+3. Document the models in a way software engineers can build and validate it,
+   and document their understanding comprehensively for future researchers
+   (including their future selves) who are faced with related modeling
+   challenges.
+
+
+What is a cause model?
+----------------------
+
+A cause model is a simplification of a cause of death or disease for the
+purposes of simulation modeling.  It is always an idealization of the messy
+complexity of reality, and is designed to be acceptable to outside experts on
+the cause, as well as be parsimonious with the data available from the GBD that
+might inform the model.
+
+
+How is a cause model incorporated into a larger model?
+------------------------------------------------------
+
+Our modular structure is designed to layer cause models into the
+entity-component system that has a demographic model.  Sometimes an
+intervention model will be layered in on top of this and directly change
+transition rates in one or more cause models.  But to date, it has been more
+common to have one or more risk factor models layered in to affect the
+incidence rates in the cause model, and then have an intervention model shift
+the risk exposure levels defined by the risk factor model.
+
+
+Focus on goal 3:
 ----------------
 
-The structure of a cause model
-------------------------------
+
+Why do we want a document that describes each cause model?
+----------------------------------------------------------
+
+* Because a lot of work goes into gaining understanding and developing an
+  appropriately complex model, and we don’t want to repeat that work.
+* Because we (researchers) need to communicate clearly and precisely with
+  software engineers and data scientists about what the model must do and what
+  data must inform it.
+* Because we will need to communicate to an outside audience, including
+  critics, how we generated substantive results of interest, and that will
+  include readers who want to know exactly how we modeled the diseases included
+  in our work.
+
+
+What does a model document look like?
+-------------------------------------
```

Comment:
We should probably spend a whole session on what the cause model template should look like and then build an rst file out of it.

Reply:
I agree. I think a cause model template document would be more helpful than this list.